pyproject.toml

```
@@ -16,6 +16,7 @@ dependencies = [
     "zarr>=2.18,<4",
     "pydantic>=2.9",
     "networkx>=3.2.1",
+    "numpy",
```

```suggestion
    "numpy>=1.26",
    "numpy>=2.1; python_version >= '3.13'",
```

it's a little annoying... but this will fix tests and ensure that users always get a version of numpy for which a wheel has been buillt